xbmc/utils/test/TestLabelFormatter.cpp

```diff
@@ -31,7 +31,7 @@ class TestLabelFormatter : public testing::Test
 TEST_F(TestLabelFormatter, FormatLabel)
 {
   XFILE::CFile *tmpfile;
-  std::string tmpfilepath, destpath;
+  std::string tmpfilepath;
```

Comment:
https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#87-variables

Use CamelCase. Type prefixing (Systems Hungarian notation) is discouraged.

Reply:
The change is specific to unusedvariable warning. No other changes are intended, or will be done in this PR